src/api/app/models/project.rb

```diff
@@ -75,6 +75,7 @@ def autocomplete(search)
 
   has_many :target_of_bs_request_actions, class_name: 'BsRequestAction', foreign_key: 'target_project_id'
   has_many :target_of_bs_requests, through: :target_of_bs_request_actions, source: :bs_request
+  has_many :required_checks, class_name: 'Status::RequiredCheck', as: :checkable
```

Comment:
It should be a serialized field too (like `Repository`), right? The class `Status::RequiredCheck` doesn't exist.

Reply:
Good catch! I did not create the serialized field in project yet as we do not have a use case for it. So far we only support repository. But the association is wrong and I will drop it.